apis/s3/src/main/java/org/jclouds/s3/blobstore/S3BlobStore.java

```diff
@@ -268,6 +268,7 @@ public String putBlob(String container, Blob blob, PutOptions overrides) {
 
       // TODO: Make use of options overrides
       PutObjectOptions options = new PutObjectOptions();
+      options.withServerSideEncryption();
```

Comment:
This should not be hardwired into the framework.  Instead you should make use of PutOptionsOverride parameter here.  JClouds people will probably refuse your change otherwise.